### DISCUSSION TITLE
How do I delete a repository that has been disabled?

### DISCUSSION THREAD

**Discussion by HardcoreGamer66** [community/community#50638] [Repositories] [answered]
### Select Topic Area

General

### Body

I recently sadly found out that a repository that I had forked, has been disabled. My fork was disabled too. I wanted to name another repository with the same name as the disabled repository.
It turns out I can't because the repository is still marked as in my account. The solution to that I thought was to delete it, but I couldn't. Thank you for taking the time to read this.

**Comment by radar-blip** [comment]
You still should be able to delete it, through the repos settings.

**Reply by HardcoreGamer66** [reply]
I can't access it through the repo's settings. The repo is taken down due to a DMCA Takedown Procedure.

**Comment by airtower-luna** [comment, marked as answer]
If you can't access the repository settings to delete it, you'll have to [contact GitHub support](https://support.github.com/request) and ask them to take care of it.

**Reply by Rabab-Muhammad** [reply]
Please help me to deleted my repository

**Comment by lauhkz** [comment]
i'm having the same problem but i can't open a ticket
